CHANGELOG.rst

```diff
@@ -12,7 +12,8 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
-*
+* Support for Python 3.5-3.8
+
```

Comment:
In this case, we need to add that we dropped support for python2.7

Reply:
Added like so!